### DISCUSSION TITLE
20차 모임 (11/15) 발표 주제 제안

### DISCUSSION THREAD

**Discussion by haksungjang** [OpenChain-KWG/meeting#4] [General]
OpenChain Korea Work Group 20차 모임이 삼성전자 서초사옥에서 11/15(수)에 예정되어 있습니다. 

이 때 주제 발표를 원하시는 분은 발표 제목과 간단한 내용을 comment로 남겨주시기 바랍니다.
혹시, (내가 발표하지는 못하지만) 누군가가 공유해주기를 원하시는 주제가 있어도 comment 남겨주세요.

감사합니다.

**Comment by arlo-ha** [comment]
카뱅에서 ISO 18974를 준비중에 있는데, 준비과정에 대해 발표를 하겠습니다.
가제: 카카오뱅크 오픈소스 보안 보증 준비 사례 공유

**Reply by haksungjang** [reply]
감사합니다!!

**Comment by yoobato** [comment]
저희 금융결제원 오픈소스 컴플라이언스 체계 구축 일정도 확정되었습니다. ('23. 11월 ~ '24. 1월 예정)
아마 내년도 첫 분기 모임에서 저희 ISO/IEC 5230:2020 인증 사례도 공유드릴 수 있을 것 같습니다. 🛩️

**Reply by haksungjang** [reply]
오, 멋지십니다. 공기업 첫 사례 아닌가요? 좋은 본보기가 될 것으로 기대합니다!!! ^^
